vpr/src/place/place.cpp

```diff
@@ -321,15 +333,18 @@ static void update_move_nets(int num_nets_affected);
 static void reset_move_nets(int num_nets_affected);
 
 static e_move_result try_swap(float t,
+                              float crit_exponent,
```

Comment:
The argument list is gettting very long. Could some of these variables be grouped in a struct and a pointer to that be passed around?